### DISCUSSION TITLE
Readme: not exact about É À etc. on Mac AZERTY (It's totally possible)

### DISCUSSION THREAD

**Discussion by juliendorra** [qwerty-fr/qwerty-fr#83] [General]
Hi, as a former + still occasional Mac AZERTY user, I wanted to point that on Mac with an AZERTY keyboard, you can type capitalized accented letters by activating caps lock, then type the key for the accented letter.

It's not 100%  discoverable and I encountered Mac AZERTY users who didn't know. It also means 3 presses if you want to then revert to lowercase letters.

But on the upside it means that when someone starts writing in CAPSLOCK MODE the letters are naturally accented.

So this impossibility might be a Windows and Linux only issue.

**Comment by devnoname120** [comment]
Could you do the research and then open a PR?

**Reply by juliendorra** [reply]
Did a pull request. Might be nitpicking, but I guess people who cares about keyboard layout are a bit obsessional 😅 (I use my own Qwerty layout https://github.com/juliendorra/QWERTY-for-French that feels just right for me 😄)
